addons/website/static/src/js/menu/content.js

```diff
@@ -645,8 +645,10 @@ var EditMenuDialog = weWidgets.Dialog.extend({
                     'to_delete': this.toDelete,
                 }
             ],
-        }).then(function () {
-            return _super();
+        }).then(() => {
+            // For the EditMenuDialog, saving the dialog should not close it,
+            // as a reload will be done to show the updated menus.
+            return _super(false);
```

Comment:
A `_super` call must always use `arguments`. You can tweak the argument list before transferring it but it should be used.

The `closeOnSave` parameter should really be a parameter of the class and not of the method IMO, either this is a dialog which requires a reload or not, it's not dependent of the caller but to the dialog itself. Doing it your way makes custo akward too: if someone use this dialog and give `closeOnSave: true`, it is just ignored as you ignore the given parameter and force `false`.

I agree it is details in this case but better code the same way systematically :wink: